source/a_box/tmu/tmu.h

```diff
@@ -27,6 +27,9 @@
 // max and min values
 #define ERROR_HIGH INT16_MAX
 #define ERROR_LOW INT16_MIN
+// max and min temp values
+#define MAX_TEMP_TMU 60
+#define MIN_TEMP_TMU 10
```

Comment:
see comments on set and get Fault, don't hardcode values here, we want to use the values from the fault config. Also wouldn't these have had to be *10 anyway? Consider converting the values you send into setFault to C ( x/10) so that when someone looks at the fault_config.json they just see normal temp values. The only negative affect is that it would decrease the resolution that you are putting in, but thats fine since our thresholds are whole numbers.

Reply:
Fixed, got rid of the hardcoded values and am using the thresholds from fault config. Didn't change the fault thresholds to x/10, but you can ofc when I push.